llvm/test/tools/llvm-objdump/XCOFF/aux-entry-invalid-type.test

```diff
@@ -54,3 +31,4 @@ Symbols:
       - Type:                   AUX_CSECT
         SymbolAlignmentAndType: 5
         StorageMappingClass:    XMC_PR
+
```

Comment:
Nit: Looks like you've added an additional blank line unnecessarily? Files should end in exactly one \n